docs/user/overview.md

```diff
@@ -47,6 +50,38 @@ The following targets can be destroyed by the installer:
 
 ### Multiple Invocations
 
-In order to allow users to customize their installation, the installer can be invoked multiple times. The state is stored in a hidden file in the target directory and contains all of the intermediate artifacts. This allows the installer to pause during the installation and wait for the user to modify intermediate artifacts.
+In order to allow users to customize their installation, the installer can be invoked multiple times. The state is stored in a hidden file in the asset directory and contains all of the intermediate artifacts. This allows the installer to pause during the installation and wait for the user to modify intermediate artifacts.
```

Comment:
Changing "target" -> "asset" here gets us closer to [the wording we use for `--dir`][1].

[1]: https://github.com/openshift/installer/blob/v0.7.0/cmd/openshift-install/main.go#L46